cv_next/server/api/cvs.ts

```diff
@@ -156,34 +156,70 @@ export async function getPaginatedCvs(
 
     const supabase = SupabaseHelper.getSupabaseInstance();
     let query = supabase
-      .from(Tables.cvs)
+      .from("cvs")
       .select("*")
-      .range(from, to - 1);
+      .order(CvKeys.created_at, { ascending: false });
+    let profileQuery = supabase.from("profiles").select("id");
 
     logger.debug(filters, "filters");
-    if (filters) {
-      if (filters.searchValue) {
-        query = query.textSearch(CvKeys.description, filters.searchValue);
-      }
-      if (filters.categoryIds) {
-        logger.debug(filters.categoryIds, "category ids");
-        query = query.overlaps(CvKeys.cv_categories, filters.categoryIds);
+
```

Comment:
Everything in this function that depends on the `searchValue` is complicated to read. There are multiple `if (filters?.searchValue)` statements, why? It'd be more readable to separate these statements to local functions with appropriate names